.github/workflows/code-review.yml

```
@@ -17,9 +17,30 @@
+        echo $(git diff --unified=3 ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }})
 
-        git diff --unified=3 ${{ github.event.before }} ${{ github.event.after }} > diff.txt
+        git diff --unified=3 ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} > diff.txt
         cat diff.txt
+
+        git_diff=$(cat diff.txt)
+        echo "DIFF=$git_diff" >> $GITHUB_ENV
+
+
+    - name: review to pull request
+      uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)